### PR TITLE
Update to 3.11.0-alpha.7 + isort rules.

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

**Comment by icemac** [comment]
Hm, a7 is not yet there, so we probably have to wait some hours.

File: src/ComputedAttribute/__init__.py [hunk 1]
```diff
@@ -1,5 +1,4 @@
-from ExtensionClass import Base
-from ExtensionClass import C_EXTENSION
+from ExtensionClass import C_EXTENSION, Base
```
Comment (dataflake) [inline src/ComputedAttribute/__init__.py:1]: The `isort` rules we have everywhere else specify one import per line.

File: src/ExtensionClass/tests.py [hunk 2]
```diff
@@ -13,12 +13,11 @@
...
 
 
-from doctest import DocTestSuite
 import sys
 import unittest
+from doctest import DocTestSuite
 
-from ExtensionClass import Base
-from ExtensionClass import ExtensionClass
+from ExtensionClass import Base, ExtensionClass
```
Comment (dataflake) [inline src/ExtensionClass/tests.py:20]: See above

File: src/MethodObject/__init__.py [hunk 3]
```diff
@@ -1,7 +1,6 @@
 from types import MethodType
 
-from ExtensionClass import Base
-from ExtensionClass import C_EXTENSION
+from ExtensionClass import C_EXTENSION, Base
```
Comment (dataflake) [inline src/MethodObject/__init__.py:3]: See above

**Review by dataflake — CHANGES_REQUESTED** [review]
The `isort` rules seem to have changed. Imports should be one per line as we do everywhere else.

File: src/ComputedAttribute/__init__.py (hunk 1 above)
Comment (icemac) [reply src/ComputedAttribute/__init__.py:1]: This is fixed in `meta/config`, I just have to re-apply it.

**Comment by icemac** [comment]
@dataflake Do you think this PR is okay now?

**Review by dataflake — APPROVED** [review]

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**